### ISSUE TITLE
Extension fails to load

### ISSUE THREAD

**Issue by matt-ayers** [microsoft/vscode-azurearmtools#1291]
Does this occur consistently? Yes
Repro steps:
1. Open ARM Template downloaded from Azure Portal in VSCode
2. Wait while VSCode tries to load ARM Template extension

Action: startArmLanguageServer
Error type: Error
Error Message: ARM Template Language Server: An error occurred starting the language server.

spawn EPERM

Version: 0.15.0
OS: win32
OS Release: 10.0.19042
Product: Visual Studio Code
Product Version: 1.55.2
Language: en

<details>
<summary>Call Stack</summary>

```
extension.bundle.js:16:243231extension.bundle.js:16:243231
rejected extension.bundle.js:16:239155
```

</details>

**Comment by StephenWeatherford** [comment]
@matt-ayers Thanks for taking the time to file this!   Can you check the Output window, select "Azure Resource Manager Tools" in the dropdown, and post the contents here (be sure to remove anything that you might not want public)? 

Thanks!

**Comment by matt-ayers** [comment]
Using dotnet core executable at c:\Users\XXXX\AppData\Roaming\Code\User\globalStorage\ms-dotnettools.vscode-dotnet-runtime\.dotnet\3.1.14\dotnet.exe
Starting ARM Template Language Server at c:\Users\XXXX\.vscode\extensions\msazurermtools.azurerm-vscode-tools-0.15.0\languageServer\Microsoft.ArmLanguageServer.dll
Language server nuget version: 3.0.0-preview.21158.1
Client options:
{
  "documentSelector": [
    {
      "language": "arm-template",
      "scheme": "file"
    },
    {
      "language": "arm-template",
      "scheme": "untitled"
    }
  ],
  "diagnosticCollectionName": "ARM Template Language Server diagnostics",
  "outputChannel": {
    "outputChannel": {
      "name": "Azure Resource Manager Tools",
      "extensionPrefix": "azureResourceManagerTools",
      "_outputChannel": {
        "name": "Azure Resource Manager Tools",
        "_channel": {}
      }
    }
  },
  "revealOutputChannelOn": 3,
  "synchronize": {
    "configurationSection": "azureResourceManagerTools"
  },
  "middleware": {}
}
Server options:
{
  "run": {
    "command": "c:\\Users\\XXXX\\AppData\\Roaming\\Code\\User\\globalStorage\\ms-dotnettools.vscode-dotnet-runtime\\.dotnet\\3.1.14\\dotnet.exe",
    "args": [
      "c:\\Users\\XXXX\\.vscode\\extensions\\msazurermtools.azurerm-vscode-tools-0.15.0\\languageServer\\Microsoft.ArmLanguageServer.dll",
      "--logLevel",
      "Warning"
    ],
    "options": {
      "shell": false
    }
  },
  "debug": {
    "command": "c:\\Users\\XXXX\\AppData\\Roaming\\Code\\User\\globalStorage\\ms-dotnettools.vscode-dotnet-runtime\\.dotnet\\3.1.14\\dotnet.exe",
    "args": [
      "c:\\Users\\XXXX\\.vscode\\extensions\\msazurermtools.azurerm-vscode-tools-0.15.0\\languageServer\\Microsoft.ArmLanguageServer.dll",
      "--logLevel",
      "Warning"
    ],
    "options": {
      "shell": false
    }
  }
}
[Error - 11:53:47] Starting client failed
Error: spawn EPERM
                at ChildProcess.spawn (internal/child_process.js:407:11)
                at Object.spawn (child_process.js:557:9)
                at c:\Users\XXXX\.vscode\extensions\msazurermtools.azurerm-vscode-tools-0.15.0\dist\node_modules\vscode-languageclient\lib\main.js:347:40
Error: ARM Template Language Server: An error occurred starting the language server.
 
spawn EPERM
 
Matt

**Comment by StephenWeatherford** [comment]
Thanks!  Unfortunately that didn't help much, and I'll be out the next 2 weeks on vacation.  I'd suggest first of all trying this:
1) Uninstall the extension
2) remove the folder it's install in:
   c:\Users\XXXX\.vscode\extensions\msazurermtools.azurerm-vscode-tools-0.15.0
3) reinstall the extension (we just released version 0.15.1).  

If that doesn't fix it, let me know, I'll figure out next steps when I get back.

Stephen

**Comment by StephenWeatherford** [comment]
Related to #1295?
